src/Wallabag/CoreBundle/Controller/ExceptionController.php

```diff
@@ -1,40 +0,0 @@
-<?php
-
-namespace Wallabag\CoreBundle\Controller;
-
-use Symfony\Bundle\TwigBundle\Controller\ExceptionController as BaseExceptionController;
-use Symfony\Component\HttpFoundation\Request;
-
-/**
- * This controller allow us to customize the error template.
- * The only modified line from the parent template is for "WallabagCoreBundle".
- */
-class ExceptionController extends BaseExceptionController
-{
-    protected function findTemplate(Request $request, $format, $code, $showException)
-    {
-        $name = $showException ? 'exception' : 'error';
-        if ($showException && 'html' === $format) {
-            $name = 'exception_full';
-        }
-
-        // For error pages, try to find a template for the specific HTTP status code and format
-        if (!$showException) {
-            $template = sprintf('@WallabagCore/Exception/%s.%s.twig', $name, $format);
```

Comment:
this was the only difference from the base one, so now the base error template is moved to use the official way of overriding a template from a bundle